README.md

```diff
@@ -44,47 +43,49 @@ directory.
 
 |    Command    | Description |
 | :-----------: | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-|  **`go-quai`**   | Our main Quai CLI client. It is the entry point into the Quai network (main-, test- or private net), capable of running as a full node (default), archive node (retaining all historical state) or a light node (retrieving data live). It can be used by other processes as a gateway into the Quai network via JSON RPC endpoints exposed on top of HTTP, WebSocket and/or IPC transports. `go-quai --help` for command line options.|
+|  **`go-quai`**   | Our main Quai CLI client. It is the entry point into the Quai network (main-, test- or private net), capable of running as a global node (all chains), base node (subset of chains), archive node (retaining all historical state), or a light node (retrieving data live). It can be used by other processes as a gateway into the Quai network via JSON RPC endpoints exposed on top of HTTP, WebSocket and/or IPC transports. `go-quai --help` for command line options.|
```

Comment:
After discussion in person, the conclusions are:
1) use slice, instead of chain or zone
2) use the names given in the table above^

**Some example names:**
global archive node
light multi-slice node
full slice node